src/averageAges.js

```diff
@@ -15,13 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mens = people.filter(person => person.sex === 'm');
+
+  if (century !== undefined) {
```

Comment:
Same as in the men's average age function, you can remove the explicit check for division by zero, it's handled with the `toFixed` and `parseFloat`.